.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  BuildAndDeploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Push backend to Dockerhub
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: dagandersen/backend
+        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
+        password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
+        workdir: backend
+    - uses: actions/checkout@master
+    - name: Push frontend to Dockerhub
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: dagandersen/frontend
+        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
+        password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
+        workdir: frontend
+
+    - uses: actions/checkout@v1
```

Comment:
Is it on purpose you use @v1 rather that @master as before here?

Reply:
I have simply copy-pasted from the template. I guess `v1` is a stable build and `master` is current state of the repo... i dont know... i dont think it matters